wagtail/wagtailcore/management/commands/move_pages.py

```diff
@@ -6,16 +6,15 @@
 
 
 class Command(BaseCommand):
-    args = "<from id> <to id>"
-
-    def handle(self, _from_id, _to_id, **options):
-        # Convert args to integers
-        from_id = int(_from_id)
-        to_id = int(_to_id)
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('from_id', nargs='?', type=int)
```

Comment:
I think you want to drop `nargs='?'` in all cases. Aren't these required arguments? `nargs='?'` makes it an optional argument defaulting to `default` (`None` in this case).